packages/amplify-provider-awscloudformation/src/commands/awscloudformation/override.ts

```diff
@@ -2,13 +2,19 @@
     entry code for amplify override root
 */
 
-import { generateOverrideSkeleton } from '../../utils/override-skeleton-generator';
+import path from 'path';
+import { generateOverrideSkeleton } from 'amplify-cli-core';
 
 const subcommand = 'override';
 
 module.exports = {
```

Comment:
we can change this similar way 
export const run = async(context: $TSContext) => {
	  const { amplify } = context;